src/main/java/seedu/address/logic/commands/AddContactToTeamCommand.java

```diff
@@ -0,0 +1,81 @@
+package seedu.address.logic.commands;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.contact.Contact;
+import seedu.address.model.team.Team;
+
+import static java.util.Objects.requireNonNull;
+
+public class AddContactToTeamCommand extends Command {
+
+    public static final String COMMAND_WORD = "add-to-team";
```

Comment:
Since all team commands start with team, I think you can change this command word to add-contact or something similar. Then an example command would be team 1 add-contact 1